packages/client/src/helpers/gtag.js

```diff
@@ -14,7 +14,10 @@
 export function gtagConfig(config) {
   // See index.html for the definition of the gtag function.
   if (typeof window.gtag === 'function' && window.APP_CONFIG?.GOOGLE_TAG_ID) {
-    window.gtag('config', window.APP_CONFIG.GOOGLE_TAG_ID, config);
+    window.gtag('config', window.APP_CONFIG.GOOGLE_TAG_ID, {
+      ...(window.APP_CONFIG?.GOOGLE_ANALYTICS_DEBUG ? { debug_mode: true } : {}),
```

Comment:
This is a bit annoying, but [according to the docs](https://support.google.com/analytics/answer/7201382#zippy=%2Cgoogle-tag-gtagjs:~:text=Disable%20debug%20mode,disable%20debug%20mode.)
> To disable debug mode, exclude the 'debug_mode' parameter; setting the parameter to `false` doesn't disable debug mode.

Therefore, we're conditionally including `debug_mode: true` rather than conditionally toggling between `true` and `false`.